### ISSUE TITLE
Not all the StatefulSet reconciled in multi AZ deployment.

### ISSUE THREAD

**Issue by ywysuibian** [cloudfoundry-incubator/quarks-statefulset#17]
**Background**
- Kubecf deployment in 2 AZs
- 2 `diego-api` StatefulSets associated to `diego-api` Quarks StatefulSet

**Describe the bug**
- Every time, only one `diego-api` StatefulSets could be reconciled. If the reconciled StatefulSet is the passive one, then the corresponding service is not available.

**To Reproduce**
as minimally and precisely as possible

**Expected behavior**
Both the 2 `diego-api` StatefulSets should be reconciled

**Environment**
 - CF-Operator Version: 4.5.6

**Additional context**
```
2020-09-27T12:44:19.855Z	DEBUG	active-passive-reconciler	quarksstatefulset/util.go:59	Listing StatefulSets owned by QuarksStatefulSet 'kubecf/diego-api' via informer
2020-09-27T12:44:19.858Z	DEBUG	active-passive-reconciler	quarksstatefulset/util.go:91	Found StatefulSet 'diego-api-z0' owned by QuarksStatefulSet 'kubecf/diego-api
2020-09-27T12:44:19.858Z	DEBUG	active-passive-reconciler	quarksstatefulset/util.go:91	Found StatefulSet 'diego-api-z1' owned by QuarksStatefulSet 'kubecf/diego-api
2020-09-27T12:44:19.858Z	DEBUG	active-passive-reconciler	quarksstatefulset/util.go:52	Getting the latest StatefulSet with version '2' owned by QuarksStatefulSet 'kubecf/diego-api
2020-09-27T12:44:19.858Z	DEBUG	active-passive-reconciler	quarksstatefulset/active_passive_reconciler.go:99	validating probe in pod: 'kubecf/diego-api-z1-0'
2020-09-27T12:44:19.859Z	DEBUG	controller-runtime.manager.events	recorder/recorder.go:52	Normal	{"object": {"kind":"QuarksStatefulSet","namespace":"kubecf","name":"diego-api","uid":"8cc509c6-3504-4280-8c8f-4e417ca93057","apiVersion":"quarks.cloudfoundry.org/v1alpha1","resourceVersion":"13311615"}, "reason": "active-passive", "message": "validating probe in pod: 'kubecf/diego-api-z1-0'"}
2020-09-27T12:44:20.352Z	DEBUG	active-passive-reconciler	quarksstatefulset/active_passive_reconciler.go:99	failed to execute active/passive probe: failed executing command in pod: 'kubecf/diego-api-z1-0', container: bbs-bbs: command terminated with exit code 1
2020-09-27T12:44:20.352Z	DEBUG	active-passive-reconciler	controller/controller.go:256	periodSeconds probe was not specified, going to default to 30 secs
2020-09-27T12:44:20.352Z	DEBUG	controller-runtime.manager.events	recorder/recorder.go:52	Normal	{"object": {"kind":"QuarksStatefulSet","namespace":"kubecf","name":"diego-api","uid":"8cc509c6-3504-4280-8c8f-4e417ca93057","apiVersion":"quarks.cloudfoundry.org/v1alpha1","resourceVersion":"13311615"}, "reason": "active-passive", "message": "failed to execute active/passive probe: failed executing command in pod: 'kubecf/diego-api-z1-0', container: bbs-bbs: command terminated with exit code 1"}
2020-09-27T12:44:20.352Z	DEBUG	controller-runtime.manager.events	recorder/recorder.go:52	Normal	{"object": {"kind":"QuarksStatefulSet","namespace":"kubecf","name":"diego-api","uid":"8cc509c6-3504-4280-8c8f-4e417ca93057","apiVersion":"quarks.cloudfoundry.org/v1alpha1","resourceVersion":"13311615"}, "reason": "active-passive", "message": "periodSeconds probe was not specified, going to default to 30 secs"}
```

**Comment by manno** [comment]
@ywysuibian was this fixed by PR #21 ?

**Comment by yangwenyanibm** [comment]
@manno yes, PR #21 fixed this issue.
